includes/hub/stores/class-abstract-event-log-item.php

```diff
@@ -56,6 +56,13 @@ abstract class Abstract_Event_Log_Item {
 	 */
 	private $timestamp;
 
+	/**
+	 * The action name associated with the Event Log Item.
+	 *
+	 * @var string
+	 */
+	private $action_name;
```

Comment:
It looks like the `$action` attribute above is not being used. I guess it was renamed in the code, but not here. It looks like we can safely rename that one instead of creating a new one

Reply:
Good catch! Updated in https://github.com/Automattic/newspack-network/pull/47/commits/991f008c9911b73a153e595945e42d6c1db6fc17